dbms/src/Flash/Coprocessor/DAGExpressionAnalyzer.cpp

```diff
@@ -685,19 +766,18 @@ std::tuple<Names, TiDB::TiDBCollators, AggregateDescriptions, ExpressionActionsP
     for (const auto & aggregation_key : aggregation_keys)
         step.required_output.push_back(aggregation_key);
 
-    source_columns = std::move(aggregated_columns);
```

Comment:
why delete this line?

Reply:
`source_columns` will be setup in `appendCopyColumnAfterAgg`

Reply:
ok